apps/styleguide/src/layout/LayoutModal.vue

```diff
@@ -66,7 +46,7 @@
 
 .modal-body {
   /** leave room for header and foot*/
-  max-height: calc(100vh - 200px);
+  max-height: calc(100vh - 200px - 28px);
```

Comment:
Thx, do you know what variable I should use?

Reply:
https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties

browser support: https://caniuse.com/?search=variables

Sass / Scss:

https://sass-lang.com/documentation/variables

Are better compatible. but requires a preprocessor.

Also an option (it's more like css +)

https://lesscss.org/

it can even be 'compiled' in the browser

Reply:
You don't really answer the question. The server side 'theme' is generated from sass, see https://github.com/molgenis/molgenis-emx2/tree/master/backend/molgenis-emx2-webapi/src/main/resources/theme

What I need specifically from you is the exact code you would use at this particular place. And it is a deep desire to keep things together so please don't go changing all kinds of files elsewhere because then we will never understand why we did the change at this elsewhere place.

Reply:
I was just giving options, but now I see you have a theme file with scss.
I'll elaborate a bit more.
By using fixed pixels for footer and header, you are going to have difficulty later on, when either the footer or header changes in size. Which will happen eventually. Then all the items that have these calculations need to be adjusted by hand.

Simplest way is to add to your theme.scss:

$header: 28px;
$footer: 200px;

.modal-body {
max-height: calc(100vh - $header - $footer);
}

---
add every style that requires header and footer here
---

But if you want it on component level, you'd need:

https://vueschool.io/articles/vuejs-tutorials/globally-load-sass-into-your-vue-js-applications/

Reply:
I decided for now to us mt-0 on dialogue so this issue doesn't occur.